src/lib/CollisionPrevention/CollisionPrevention.cpp

```diff
@@ -69,104 +65,83 @@ bool CollisionPrevention::initializeSubscriptions(SubscriptionArray &subscriptio
 	return true;
 }
 
-void CollisionPrevention::reset_constraints()
-{
-
-	_move_constraints_x_normalized.zero();  //normalized constraint in x-direction
-	_move_constraints_y_normalized.zero();  //normalized constraint in y-direction
-
-	_move_constraints_x.zero();  //constraint in x-direction
-	_move_constraints_y.zero();  //constraint in y-direction
-}
-
-void CollisionPrevention::publish_constraints(const Vector2f &original_setpoint, const Vector2f &adapted_setpoint)
+void CollisionPrevention::calculate_constrained_setpoint(const float max_acc, Vector2f &setpoint)
```

Comment:
It seems we have some inconsistency, sometimes we are having output parameters first, and sometimes last. Is there a standard way of doing it?